### DISCUSSION TITLE
Parts

### DISCUSSION THREAD

**Discussion by glazet** [glazet/QuadEVCharger#3] [General]
Here can be our repository of possible parts and materials to be used in the project.

**Comment by neilh10** [comment]
I came across these EV relays 
https://www.azettler.com/ev-charging-relays/
https://www.azettler.com/pdfs/az2800.pdf   or AZ2850
https://www.digikey.com/en/products/detail/american-zettler/AZ2800-2A-12DE/12171621
https://www.azettler.com/pdfs/azev200.pdf
https://www.digikey.com/en/products/filter/power-relays-over-2-amps/188?s=N4IgjCBcoLQExVAYygFwE4FcCmAaEA9lANogCsIAugL7X4KSkCCAWgKIBqcADN1fgDZEIAJYATKCBhhuEfAAdUkkPlQBPedkkBDAM4paQA 

The relays seem better pricing than the associated Solid State Relay 2 "Form A"
Mouser SSR 2 FORM A : 
RA2A48D40 $85.50 Solid State Relays - Industrial Mount SSR 2 POLE ZS 480V 40A RES
Dual Channel, Control Voltage 4-32VDC  https://www.sensata.com/sites/default/files/a/sensata-evolution-dual-series-panel-mount-datasheet.pdf 
RKD2A23D50P $81.7@100  SSR RELAY SPST-NO 50A 24-265V
PCS28-12D-240A-40R  $35.83 SSR RELAY SPST-NO 40A 48-280V
Diagram https://www.digikey.com/en/products/detail/sensata-crydom/CC2450W3VR/2595829
PCS28-12D-240A-40R $35.83 ~

**Comment by glazet** [comment]
This Raspberry PICO could useful for the communication through the pilot control pin. It is capable of outputting a 1kHz square wave signal but at a low voltage. Will probably need an amplifier of some sort.

https://www.digikey.com/en/products/detail/raspberry-pi/SC0915/13684020?utm_adgroup=&utm_source=google&utm_medium=cpc&utm_campaign=PMax%20Shopping_Product_Low%20ROAS%20Categories&utm_term=&utm_content=&gclid=Cj0KCQjwx5qoBhDyARIsAPbMagDgnTaGq54GIWpzjvpuJJi5n0RqvsgurPDvfagRQrDfqGnbt6NgH1YaAhU5EALw_wcB

**Comment by CalvinJDP** [comment]
I came across a few connectors we could potentially use. 

- This is an inexpensive connector, unfortunately only goes up to 32 Amps. We can keep this in mind in case we cant find a cheap connector.
[Link 1](https://www.nocoev.com/product/SAE_J1772_16A_AC_Charging_Gun.htm#gsc.tab=0)


- Another connector we could use however this one is relatively expensive. It is an extension cable, but we can probably splice it.
[Link 2](https://qccharge.com/products/jlong)

**Comment by djangodemetri** [comment]
[4 Channel 16bit ADC for RPi with Amplifier](https://www.adafruit.com/product/1085)
[220VAC to 5VDC Inverter](https://www.trcelectronics.com/View/Mean-Well/RS-15-5.shtml?utm_term=&utm_campaign=Sales-Performance+Max-4&utm_source=adwords&utm_medium=ppc&hsa_acc=9805199880&hsa_cam=20355717806&hsa_grp=&hsa_ad=&hsa_src=x&hsa_tgt=&hsa_kw=&hsa_mt=&hsa_net=adwords&hsa_ver=3&gad=1&gclid=CjwKCAjwsKqoBhBPEiwALrrqiFKP6I0N0dge6n3RBVmwzYIZDAeKy1e_KwkfBiksdn9E6nL5I6DwRxoCPtgQAvD_BwE)

**Comment by glazet** [comment]
Here is our google sheet of potential parts we plan on getting. This is where a lot of this weeks work has been done:
https://docs.google.com/spreadsheets/d/10eQG6eU977sr-Vo4AyLgy35bYP38K_eaPXTxfUFYXaI/edit?usp=sharing
